src/configs/directory.rs

```diff
@@ -22,6 +24,15 @@ impl<'a> RootModuleConfig<'a> for DirectoryConfig<'a> {
             fish_style_pwd_dir_length: 0,
             use_logical_path: true,
             prefix: "in ",
+            separator: match std::path::MAIN_SEPARATOR {
```

Comment:
**question:**
What does this `match` do that a `char.to_string()` wouldn't do?

Reply:
see my earlier comments

Reply:
How about `str::utf8` as an alternative solution:
```suggestion
            separator: std::str::from_utf8(&[std::path::MAIN_SEPARATOR as u8]).unwrap(),
```

Reply:
Given that `std::path::MAIN_SEPARATOR` is `u32` should `try_into` be used here? Also I don't think all single byte utf8 chars map one-to-one to `u32` Unicode scalar values.